machine.go

```diff
@@ -20,6 +20,7 @@ type machine struct {
 	Type        string     `json:"-"`
 	Fingerprint string     `json:"fingerprint"`
 	Hostname    string     `json:"hostname"`
+	Ip          string     `json:"ip"`
```

Comment:
to be clear: I'm using offline validation and checkout the machine.lic file in the web portal. It contains the ip data, I've verified. With this patch I can access it, otherwise I can't.

Reply:
Thanks for sharing more info about your use case. Your PR did not include this context in the description, so it was hard to get the full picture of what you were wanting to do here. Ideally, we'd have a unified IP attribute that is set via the SDK during activation, but I understand in your case, the machine is created via the dashboard and not the SDK.

I need to look at the failing tests (looks unrelated to the PR) and I'll consider merging.

Reply:
Yeah, that was unfortunate - I wanted to add the description, then received a phone call and totally forgot about it.